app/components/domains-table.tsx

```diff
@@ -89,6 +90,25 @@ export default function DomainsTable(props: DomainsTableProps) {
     }
   }
 
+  function renderDomainName(domainName: string) {
```

Comment:
Would you mind renaming the function to `renderDnsRecordName`, since it's being added to this PR? I have to standardize the naming convention as specified #245 for the existing code.